pkg/render/node.go

```diff
@@ -624,6 +639,9 @@ func (c *nodeComponent) flexVolumeContainer() v1.Container {
 		Name:         "flexvol-driver",
 		Image:        components.GetReference(components.ComponentFlexVolume, c.cr.Spec.Registry, c.cr.Spec.ImagePath),
 		VolumeMounts: flexVolumeMounts,
+		SecurityContext: &v1.SecurityContext{
```

Comment:
I think we want to add this but why not add it to the CNI container also?

Reply:
I missed it oops. :P